package.json

```diff
@@ -50,10 +50,10 @@
     "build-pages": "concurrently \"npm run build-pages:ts\" \"npm run build-pages:less\" && npm run build-pages:eleventy",
     "build-pages:ts": "cd pages && webpack",
     "build-pages:less": "lessc pages/src/localdev.less pages/static/bundles/localdev.css --source-map=pages/static/bundles/localdev.css.map",
-    "build-pages:eleventy": "eleventy",
+    "build-pages:eleventy": "ELEVENTY_ENV=production npx @11ty/eleventy",
```

Comment:
Is it working in all OS?
It seems like I have a problem with running it in such way on WIndows
Mb it has an option to pass environment as a regular command-line argument?